test/unit/test_hpe3par_snapshot.py

```diff
@@ -190,7 +206,10 @@ def test_main_exit_absent(self, mock_delete_snapshot, mock_module, mock_client):
             'allow_remote_copy_parent': None,
             'new_name': None,
             'rm_exp_time': None,
-            'state': 'absent'
+            'state': 'absent',
```

Comment:
This is not clear to me? what exactly are we trying here?

Reply:
'absent' state mean deleting snapshot, in the same way 'present' state means creating snapshot.